src/services/jsDoc.ts

```diff
@@ -106,15 +102,19 @@ namespace ts.JsDoc {
                 const { name } = tag as JSDocTypedefTag | JSDocPropertyTag | JSDocParameterTag;
                 return name ? withNode(name) : comment;
             default:
-                return comment;
+                return comment || "";

```

Comment:
adding `|| ""` goes the opposite direction of adding `| undefined` to the return type. Is the latter still correct?

Reply:
Whoops, shouldn't have left this in.